File: template/Go/fizz-buzz.go
package main

import (
	"fmt"
    "strconv"
)

func FizzBuzz(i int) string{
  if i % 3 == 0 && i % 5 == 0 {
    return "FizzBuzz"
  } else if i % 3 == 0 {
    return "Fizz"
  } else if i % 5 == 0 {
    return "Buzz"
  }
  return strconv.Itoa(i)
}
func main() {
    for i := 1 ; i <= FIZZBUZZ_CYCLES; i++ {
        fmt.Println(FizzBuzz(i));
    }
}

File: template/Dart/fizz-buzz.dart
void fizzBuzz(int foo) {
  if (foo % 15 == 0) {
    print("FizzBuzz");
  } else if (foo % 5 == 0) {
    print("Buzz");
  } else if (foo % 3 == 0) {
    print("Fizz");
  } else {
    print(foo);
  }
}

void main() {
  for (var i = 1; i <= FIZZBUZZ_CYCLES; i++) {
    fizzBuzz(i);
  }
}
